script.js

```diff
@@ -0,0 +1,746 @@
+        regionColor: "#004529"
+    },
+    AR: {
+        selLanes: { lanes:true, paths:false, trails:false },
+        selBuffers: { ft500:false, ft1000:false, ft2500:false, ft5280:false },
+        laneData:[],
+        laneLayers:[],
+        bufferLayers: { ft500:null, ft1000:null, ft2500:null, ft5280:null },
+        bikeLaneStyle: { color: "#006837", weight: 2 },
+        regionColor: "#006837"
+    },
+    DC: {
+        selLanes: { lanes:true, paths:true, trails:true },
+        selBuffers: { ft500:false, ft1000:false, ft2500:false, ft5280:false },
+        laneData:[],
+        laneLayers:[],
+        bufferLayers: { ft500:null, ft1000:null, ft2500:null, ft5280:null },
+        bikeLaneStyle: { color: "#238443", weight: 2 },
+        regionColor: "#238443"
+    },
+    MO: {
+        selLanes: { lanes:true, paths:false, trails:false },
+        selBuffers: { ft500:false, ft1000:false, ft2500:false, ft5280:false },
+        laneData:[],
+        laneLayers:[],
+        bufferLayers: { ft500:null, ft1000:null, ft2500:null, ft5280:null },
+        bikeLaneStyle: { color: "#41ab5d", weight: 2 },
+        regionColor: "#41ab5d"
+    },
+    PG: {
+        selLanes: { lanes:false, paths:false, trails:false },
+        selBuffers: { ft500:false, ft1000:false, ft2500:false, ft5280:false },
+        laneData:[],
+        laneLayers:[],
+        bufferLayers: { ft500:null, ft1000:null, ft2500:null, ft5280:null },
+        bikeLaneStyle: { color: "#78c679", weight: 2 },
+        regionColor: "#78c679"
+    }
+}
+
+// ======= regions =======
+var regions = {
+    AL: {
+        id: "AL_4",
+        name: "Alexandria",
+        box: { NW: [null, null], SE: [null, null] },
+        center: [null, null],
+        zoneFiles: [],
+        laneFiles: {
+            lanes: "VA_Alexandria_Bike.geojson",
+            paths: null,
+            trails: null },
+        bufferFiles: {
+            ft500:"VA_Alexandria_Bike_Buffer_500ft.geojson",
+            ft1000:"VA_Alexandria_Bike_Buffer_1000ft.geojson",
+            ft2500:"VA_Alexandria_Bike_Buffer_2500ft.geojson",
+            ft5280:"VA_Alexandria_Bike_Buffer_5280ft.geojson" }
+    },
+    AR: {
+        id: "AR_3",
+        name: "Arlington",
+        box: { NW: [null, null], SE: [null, null] },
+        center: [null, null],
+        zoneFiles: [],
+        laneFiles: {
+            lanes: "VA_Arlington_Bike.geojson",
+            paths: null,
+            trails: null },
+        bufferFiles: {
+            ft500:"VA_Arlington_Bike_Buffer_500ft.geojson",
+            ft1000:"VA_Arlington_Bike_Buffer_1000ft.geojson",
+            ft2500:"VA_Arlington_Bike_Buffer_2500ft.geojson",
+            ft5280:"VA_Arlington_Bike_Buffer_5280ft.geojson" }
+    },
+    DC: {
+        id: "DC_0",
+        name: "District of Columbia",
+        box: { NW: [null, null], SE: [null, null] },
+        center: [38.91, -77.04],
+        zoneFiles: ["District_Mask.geojson", "Ward__2012.geojson"],
```

Comment:
These are geographic boundaries (DC zoneFiles are wards and city boundary line; these could also be counties, etc. depending on objectives...)